### PR TITLE
feat: custom columns

### PR DESCRIPTION
**Pull request by tekumara**
Show a custom set of columns, eg:

```
ec2 describe -c SubnetId,Name
```

This allows display of columns outside the default set of InstanceId,State,Name,Type,DnsName,LaunchTime,ImageId

### CODE REVIEW

File: docs/ec2.md [hunk 1]
```diff
@@ -80,6 +84,12 @@ Show running instances sorted by date started (ie: LaunchTime), oldest first:
 ec2 describe -r -s LaunchTime
```
Comment (mhasanbulli) [inline docs/ec2.md:84]: Nit: I think several example commands are missing `aec` command at the beginning.

File: src/aec/main.py [hunk 2]
```diff
@@ -53,6 +53,7 @@ def tag_arg_checker(tag: str) -> str:
         Arg("-r", "--show-running-only", action='store_true', help="Show running or pending instances only"),
         Arg("-it", "--include-terminated", action='store_true', help="Include terminated instances"),
         Arg("-s", "--sort-by", type=str, help="Sort by one or more fields", default="State,Name"),
+        Arg("-c", "--columns", type=str, help="Customise the columns shown.", default="InstanceId,State,Name,Type,DnsName,LaunchTime,ImageId"),
```
Comment (mhasanbulli) [inline src/aec/main.py:56]: Nit
```suggestion
        Arg("-c", "--columns", type=str, help="Customise the columns shown", default="InstanceId,State,Name,Type,DnsName,LaunchTime,ImageId"),
```

**Review by mhasanbulli — APPROVED** [review]
🚀

File: docs/ec2.md (hunk 1 above)
Comment (tekumara) [reply docs/ec2.md]: good catch! that assumes the `ec2` alias, which it probably shouldn't...

**Pull request merged**